views/components/BarGraph/index.jsx

```diff
@@ -36,7 +36,7 @@ const BarGraph = ({ graph, classes }) => {
           height={50}
           margin={{ top: 0, left: 0, right: 0, bottom: 20 }}
         />
-        <XAxis dataKey="name" height={100}>
+        <XAxis dataKey="name" height={100} angle={55} textAnchor="start">
```

Comment:
🧑‍🎓  what does these do?

Reply:
The angle property rotates the label and the textAnchor aligns the label text by start, end or center. Center is the default setting, but that setting places half of the label value in the chart

Reply:
ah, looking at the screenshot that makes perfect sense 😄